shell/packages/sandstorm-db/db.js

```diff
@@ -430,6 +432,29 @@ ApiTokens = new Mongo.Collection("apiTokens");
 //   ...
 // }
 
+ApiHosts = new Mongo.Collection("apiHosts");
```

Comment:
This is a separate collection so that clients (which basically never care about this collection) don't have to load the data associated with the associated ApiToken?